changelogs/fragments/11245-monit-enum.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - monit - internal state was not reflecting when operation is "pending" in ``monit`` (https://github.com/ansible-collections/community.general/pull/11245).
```

Comment:
This is likely something we should also fix in stable-11. If you have time, can you create a backport that only contains that fix (and this part of the changelog fragment)?